package.json

```diff
@@ -115,14 +79,49 @@
     "scratch-storage": "1.3.3",
     "scratch-svg-renderer": "0.2.0-prerelease.20200610220938",
     "scratch-vm": "0.2.0-prerelease.20200820211625",
+    "style-loader": "^0.23.0",
+    "to-style": "1.3.3",
+    "wav-encoder": "1.3.0"
+  },
+  "peerDependencies": {
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/plugin-transform-async-to-generator": "^7.1.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.0.4",
+    "base64-loader": "1.0.0",
+    "chromedriver": "84.0.1",
+    "enzyme": "^3.5.0",
+    "enzyme-adapter-react-16": "1.3.0",
+    "eslint": "^5.0.1",
+    "eslint-config-scratch": "^5.0.0",
+    "eslint-import-resolver-webpack": "^0.11.1",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jest": "^22.14.1",
+    "eslint-plugin-react": "^7.12.4",
+    "gh-pages": "github:rschamp/gh-pages#publish-branch-to-subfolder",
+    "html-webpack-plugin": "^3.2.0",
+    "jest": "^21.0.0",
+    "jest-junit": "^7.0.0",
+    "lodash.throttle": "4.0.1",
+    "mkdirp": "^1.0.3",
+    "raf": "^3.4.0",
+    "react-test-renderer": "16.2.0",
+    "redux-mock-store": "^1.2.3",
+    "rimraf": "^2.6.1",
     "selenium-webdriver": "3.6.0",
     "startaudiocontext": "1.2.1",
-    "style-loader": "^0.23.0",
-    "svg-to-image": "1.1.3",
     "text-encoding": "0.7.0",
```

Comment:
While not used by Scratch Desktop (I think), default-project/index.js uses text-encoding. Also, why not change to fastestsmallesttextencoderdecoder?

Reply:
It might be a good idea to switch to smallestfastesttextencoderdecoder but I'd rather not make that part of this change.